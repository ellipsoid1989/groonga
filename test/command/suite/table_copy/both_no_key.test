#@require-input-type json

table_create Logs TABLE_NO_KEY

load --table Logs
[
{},
{},
{}
]

delete Logs --id 2

table_create ApplicationLogs TABLE_NO_KEY

table_copy Logs ApplicationLogs

dump
