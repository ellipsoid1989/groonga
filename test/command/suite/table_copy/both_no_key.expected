table_create Logs TABLE_NO_KEY
[[0,0.0,0.0],true]
load --table Logs
[
{},
{},
{}
]
[[0,0.0,0.0],3]
delete Logs --id 2
[[0,0.0,0.0],true]
table_create ApplicationLogs TABLE_NO_KEY
[[0,0.0,0.0],true]
table_copy Logs ApplicationLogs
[[0,0.0,0.0],true]
dump
table_create ApplicationLogs TABLE_NO_KEY

table_create Logs TABLE_NO_KEY

load --table ApplicationLogs
[
["_id"],
[1],
[],
[3]
]
delete --table ApplicationLogs --id 2

load --table Logs
[
["_id"],
[1],
[],
[3]
]
delete --table Logs --id 2
